docs/PostgreSQL.md

```diff
@@ -232,6 +232,23 @@ To activate this feature, do one of the following:
 wal-g backup-push /path --rating-composer
 ```
 
+#### Simple composer mode
+
+In the simple composer mode, WAL-G does not track metadata of the files backed up. This significantly reduces the memory usage on instances with `> 100k` files.
+
+Limitations or drawbacks in using Simple compose mode is,
```

Comment:
You can add an explicit note that simple mode does not support delta and catchup backups.

Also, currently, nothing prevents the user from making a delta or catchup backup with simple mode enabled, without any noticeable error. WAL-G should fail in this case and forbid creating a delta/catchup backup when the simple mode is enabled.